book/src/main/resources/static/src/pages/BookDetail.tsx

```
@@ -0,0 +1,59 @@
+    <div>
+      <button onClick={() => window.history.back()} style={{marginBottom: 16, padding: '6px 16px', borderRadius: 4, border: '1px solid #1976d2', background: '#1976d2', color: '#fff', cursor: 'pointer'}}>← 뒤로 가기</button>
+      <h2>도서 상세</h2>
+      <div style={{display: 'flex', gap: '20px', marginTop: '20px'}}>
+        {book.image && (
+          <div>
+            <img src={book.image} alt={book.title} style={{width: '200px', height: 'auto', border: '1px solid #ddd', borderRadius: '8px'}} />
+          </div>
+        )}
+        <div style={{flex: 1}}>
+          <p><strong>제목:</strong> <span dangerouslySetInnerHTML={{__html: book.title}} /></p>
+          <p><strong>저자:</strong> <span dangerouslySetInnerHTML={{__html: book.author}} /></p>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)